controller/array_action/array_mediator_svc.py

```diff
@@ -743,19 +744,19 @@ def get_host_by_host_identifiers(self, initiators):
                 nvme_host = host.name
                 connectivity_types.add(config.NVME_OVER_FC_CONNECTIVITY_TYPE)
                 logger.debug("found nvme nqn in list : {0} for host : "
-                             "{1}".format(initiators.nvme_nqn, nvme_host))
+                             "{1}".format(initiators._nvme_nqns, nvme_host))
```

Comment:
as per our conversation, in order to really make these private, I think we should either log the entire object, or move the specific initiators logging into the match methods